.pre-commit-config.yaml

```diff
@@ -90,6 +90,7 @@ repos:
         additional_dependencies:
           - numpy
           - torch>=2.6.0
+          - pytorch-finufft
```

Comment:
this will only install cpu version.

consider adding cuda optional dependencies?

Reply:
and give a better exception if used without cufionufft installed on a cuda tensor